k8s/client.go

```diff
@@ -664,47 +683,51 @@ func (client *Client) DeployCoreOperatorSync(ctx context.Context, coreCloudURL,
 		},
 		{
 			Name:  "NO_TLS_VERIFY",
-			Value: strconv.FormatBool(noTLSVerify),
+			Value: strconv.FormatBool(!params.NoTLSVerify),
```

Comment:
You moved the logic down but its the same. It's confusing why the logic is inverted while the variables are called the same...

Reply:
i can't remember why that's why i'm careful

Reply:
let's leave it as is because we know it is working now